### PR TITLE
fix LibraryIdentifier paths when initialization starts from an inline script in a subfolder

### PR DESCRIPTION
**Pull request by jakemac53**

### CODE REVIEW

File: lib/src/mirror_loader.dart [hunk 1]
```diff
@@ -183,7 +183,9 @@ class InitializationCrawler {
         }
         if (uri.scheme == 'file' || uri.scheme.startsWith('http')) {
           filePath = path.url.relative(uri.path,
-              from: path.url.dirname(_root.uri.path));
+              from: _root.uri.path.endsWith('/')
```
Comment (sigmundch) [inline lib/src/mirror_loader.dart]: not sure I follow, what is the root uri in that case?

Comment (jakemac53) [inline lib/src/mirror_loader.dart]: So for inline scripts the root uri is just the folder name, with a trailing `/`.

Comment (sigmundch) [inline lib/src/mirror_loader.dart]: oh, I thought it was the .html file name :-/, I see

**Comment by sigmundch** [comment]
lgtm

**Comment by jakemac53** [comment]
For the root library it seems to be the folder, and for inline scripts in
html imports its the .html file name

**Pull request merged**